app/services/search/search_n.rb

```diff
@@ -2,7 +2,7 @@ module Search
   class SearchN < Base
     def initialize(collections, num_results: 10, traceable: nil, include_irrelevant: false)
       @collections = collections
-      @num_results = num_results
+      @num_results = num_results || 10
```

Comment:
10 is already the default; do you need this?

Reply:
I ran into an issue where there was a Setting was being passed in, but it was blank. So the default in the method declaration didn't kick in. It was due to bad data in the DB, so maybe should let it fail?

Reply:
Ah, right.

If you want it to be valid to explicitly pass `nil` to this method and still get the default value, then I'd drop the param default and use the default inside the constructor.

I guess if you _also_ want to not pass the arg at all, you could default `num_results` to `nil` - then you don't need to provide, and if you don't the default is used. But you can provide it as `nil`, and the default is also used.